src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -68,6 +68,19 @@
                                 </p>
                             </div>
                         </c:when>
+                        <c:when test="${isAccountRetired}">
+                            <script>
+                                sendEvent('Authorization', 'Error', 'Account is retired, stale user has been sent registration');
```

Comment:
do we even use/need these events at all?

Reply:
@dfourn looks like it is for google analytics. Not sure if it is being used by anyone.